frontend/simple/js/state.js

```diff
@@ -17,7 +16,15 @@ import debounce from 'lodash/debounce'
 Vue.use(Vuex)
 var store // this is set and made the default export at the bottom of the file.
           // we have it declared here to make it accessible in mutations
-
+// Used by the getters and mutations to find the mailbox contracts
+function getMailbox (state) {
+  for (let [key, value] of Object.entries(state.contracts)) {
+    if (value === 'MailboxContract') {
+      return key
+    }
+  }
+  return null
+}
```

Comment:
This may also not be a very future-proof way of getting the mailbox... so at the very least add a `NOTE:` or a `TODO:` comment here to that effect.

```
// TODO: If we a user is ever subscribed to multiple mailboxes, this will have to be rewritten
```